index.php

```diff
@@ -55,6 +55,7 @@
 	$showInactive = filter_input(INPUT_GET, 'show_inactive', FILTER_VALIDATE_BOOLEAN);
 	$action = array_key_exists('action', $params) ? $params['action'] : null;
 	$bridge = array_key_exists('bridge', $params) ? $params['bridge'] : null;
+	$url = array_key_exists('url', $params) ? $params['url'] : null;
```

Comment:
The name of this parameter conveys no meaning. Something like `targetURL` would improve readability considerably.
_Notice_: I'm speaking about `$url`, not the query parameter!

```suggestion
	$targetURL = array_key_exists('url', $params) ? $params['url'] : null;
```

or maybe

```suggestion
	$targetURL = array_key_exists('target', $params) ? $params['target'] : null;
```

Reply:
This parameter should probably be parsed inside `$action === 'detect'`. What do you think?

Reply:
Yeah $targetURL looks way better, thanks!

I'll move the parameter parsing into the detect section as well, makes more sense there!